### PR TITLE
More Speech Verbs

### PR DESCRIPTION
**Pull request by Nairodian**
## About the PR
Adds two new suffix verbs, mumbling and stuttering, caused by a ".." and "-" respectfully.
Adds several new speech verbs for each species and other mobs, averaging out at around three per species. List is as follows;
- Humans and dwarves; say.
- Arachnids; chitter, chirp, click, hiss.
- Moths; flutter, flap, buzz, chitter.
- Slimes; slosh, burble, ooze.
- Diona; rustle, sway, creak.
- Reptiles; hiss, snort, huff, chirp.
- Robotics; state, beep.
- Skeletons; rattle, clack, gnash.
- Small mobs; squeak, piep.
- Large mobs; roar, growl.
- Monkeys; chimper, screech.
- Canines; bark, woof, howl.
- Ghosts (revenant only); complain, breathe, hum, mutter, mumble.

Also goes through and adds the existing speech verbs to a lot of new mobs that makes sense for them and were playable by normal means, things like borgs, corgis, spiders, carps, ratkings, etc.

## Why / Balance
Varity is fucking cool, adds a bit more spice which is rather quite nice.

## Media
image
- [X] I have added screenshots/videos to this PR showcasing its changes ingame.

**Changelog**
:cl:
- add: Added two new types of speech verbs, mumbling and stuttering. Add the suffix '..' or '-' respectfully, to the end of your sentences to use them.
- add: Added several more species specific speech verbs. And grants them to a whole slew of new mobs.

### CODE REVIEW

**Comment by Endecc** [comment]
Upfront, I'll say I really like the addition of mumble and stutter affix verbs, and adding new verbs to other mobs. Fits very well in my eyes.

But I'm not so much a fan of injecting randomness into basic verbs by having multiple for normal speaking. At least, assuming I'm correct in understanding what this PR is doing.
I think there's value in a consistent reading experience, giving only one verb per affix, or lack thereof in this case.

**Comment by arimah** [comment]
I have a few opinions I would like to share on this matter.

Firstly, I find randomness to be more disruptive than helpful. The varying length of each speech verb means lines of text from the same person will no longer line up nicely, making it more difficult to follow along in conversation as your eyes dart back and forth trying to find the start of the utterance.

Secondly, I find the specific choice of new suffix verbs to be... poor, honestly. I don't at all see "..." as implying a mumble – it could just as well be the character trailing off, or indicating a pause to think. Likewise, "-" does not register exclusively as a stutter to me, but also an abrupt brea—

Thirdly... well. You may remember from English class (or whatever your native language is), teachers like to encourage students to vary up 'speech verbs' whenever a character speaks. Says, states, speaks, utters, shouts, whimpers, asks, queries, wonders, snorts, chortles, chirps, and so on and so forth. Yet whenever you read professionally published literature by accomplished authors, the by far more common verb, accounting for probably over 90% of all speech verbs, is "says". Why? Because what is said matters more. You are supposed to pay more attention to the words than the verb describing the manner in which they are delivered. The verb changes only to convey information that word choice and punctuation alone cannot capture, such as when a character is, say, whispering.

All this to say: my personal opinion, for what it's worth, is that maybe we don't need all these speech verbs to spice up a game which, by its very nature, is already pretty spicy. They're kinda distracting, imo.

**Comment by Nairodian** [comment]
For the mumbling and stuttering, that's a fair point. Intention is a little hard to root out, however would "trails"/"trails off" and "stumbles" work better for the two?

As for the random verbs, I personally don't have issues with them for a few reasons. People rarely talk in long streams of sentences without someone else breaking in. There's only three-four to be picked from so a lot of the time you'll roll the same one. And if you need to follow someone's side of a conversation your eyes will likely be darting to the bold name, and then the start of the quotation already.

I tested a few actual examples of conversations I've recorded, and I don't have many issues with it myself.
image

image

image

> I could see this being weird if a long message gets fragmented with different verbs, tho.

Now I'm not really sure what this means, but I tested and long messages don't get split into different verbs in the text box, and also along side it a non-natural message.
image

image

File: Resources/Locale/en-US/chat/managers/chat-manager.ftl [hunk 1]
```diff
@@ -52,28 +52,65 @@ chat-manager-admin-channel-name = ADMIN
...
 
 chat-speech-verb-default = says
 chat-speech-verb-exclamation = exclaims
 chat-speech-verb-exclamation-strong = yells
 chat-speech-verb-question = asks
+chat-speech-verb-stutter = stutters
+chat-speech-verb-mumble = mumbles
+
+chat-speech-verb-basic-1 = speaks
+chat-speech-verb-basic-2 = utters
```
Comment (mirrorcult) [inline Resources/Locale/en-US/chat/managers/chat-manager.ftl:66]: remove `speaks` and `utters` i dont like them over just `says`

File: Resources/Locale/en-US/chat/managers/chat-manager.ftl [hunk 2]
```diff
@@ -52,28 +52,65 @@ chat-manager-admin-channel-name = ADMIN
...
-chat-speech-verb-slime = chirps
+chat-speech-verb-plant-1 = rustles
+chat-speech-verb-plant-2 = sways
+chat-speech-verb-plant-3 = creaks
 
-chat-speech-verb-robotic = states
-chat-speech-verb-reptilian = hisses
+chat-speech-verb-robotic-1 = states
+chat-speech-verb-robotic-2 = beeps
+chat-speech-verb-robotic-3 = reports
```
Comment (mirrorcult) [inline Resources/Locale/en-US/chat/managers/chat-manager.ftl:86]: remove `reports`, dont think it makes that much sense usually

**Review by mirrorcult — CHANGES_REQUESTED** [review]
im ok with variable length for sayverbs since entity name length is going to be fixed anyway, not that much variances

stutter/mumble are good i think

File: Resources/Locale/en-US/chat/managers/chat-manager.ftl (hunk 1 above)
Comment (Nairodian) [reply Resources/Locale/en-US/chat/managers/chat-manager.ftl]: Sure, done.

File: Resources/Locale/en-US/chat/managers/chat-manager.ftl (hunk 2 above)
Comment (Nairodian) [reply Resources/Locale/en-US/chat/managers/chat-manager.ftl]: Sure, done.

**Comment by Nairodian** [comment]
Oh shit- didn't even notice #19715. I _**just**_ did that for this PR, shouldn't fuck anything up I think though.

**Review by mirrorcult — APPROVED** [review]

**Pull request merged**